app/portainer/views/home/homeController.js

```diff
@@ -1,6 +1,6 @@
 angular.module('portainer.app')
-  .controller('HomeController', ['$q', '$scope', '$state', '$interval', 'Authentication', 'EndpointService', 'EndpointHelper', 'GroupService', 'Notifications', 'EndpointProvider', 'StateManager', 'LegacyExtensionManager', 'ModalService', 'MotdService', 'SystemService',
-    function($q, $scope, $state, $interval, Authentication, EndpointService, EndpointHelper, GroupService, Notifications, EndpointProvider, StateManager, LegacyExtensionManager, ModalService, MotdService, SystemService) {
+  .controller('HomeController', 
+    function($q, $scope, $state, TagService, Authentication, EndpointService, EndpointHelper, GroupService, Notifications, EndpointProvider, StateManager, LegacyExtensionManager, ModalService, MotdService, SystemService) {
```

Comment:
**Simple comment**
Nice use of `babel-plugin-angularjs-annotate` auto detection !

Reply:
yes, I realized that adding to both array and arguments would look like a lot of changes anyway :D

Reply:
thanks